src/clearTasks.js

```diff
@@ -0,0 +1,23 @@
+import { displayTasksFromLS } from './add&delete'; // eslint-disable-line import/no-cycle
+
+const clearTask = (tasks, todoList) => {
+  document.querySelector('#clr-btn').addEventListener('click', () => {
+    for (let i = 0; i < tasks.length; i += 1) {
+      if ((tasks[i].completed) === true) {
+        tasks.splice(i, 1);
+      }
+    }
```

Comment:
- [ ] Please see my previous comment. this is where the reviewer wants you to use the `filter` function. inside the `clearTask` method.